tensorflow/core/platform/default/stacktrace.h

```diff
@@ -38,8 +38,8 @@ namespace tensorflow {
 
 // Function to create a pretty stacktrace.
 inline std::string CurrentStackTrace() {
-#if defined(TF_GENERATE_BACKTRACE)
   std::stringstream ss("");
+#if defined(TF_GENERATE_BACKTRACE)
```

Comment:
Constructing `std::stringstream` and calling `ss.str()` can be expensive. How about this:

```
#if defined(TF_GENERATE_BACKTRACE)
  std::stringstream ss("");
  return ss.str();
#else
  return std::string();
#endif
```

Reply:
Sounds good! Will fix, thanks!